src/main/java/com/pubnub/api/models/consumer/history/HistoryMessageType.java

```diff
@@ -0,0 +1,22 @@
+package com.pubnub.api.models.consumer.history;
+
+import com.pubnub.api.PubNubException;
+
+public enum HistoryMessageType {
+    Message,
```

Comment:
I would change constant value to uppercase i.e. Message to MESSAGE and File to FILE.
I know that in kotlin we have them like this but I guess uppercase for constant values in enum is common standard.

We could create story to change them in kotlin as well and release in one of breaking change release?

Reply:
yeah, I we should probably change both to uppercase